charts/common/templates/lib/controller/_container.tpl

```diff
@@ -5,6 +5,9 @@ The main container included in the controller.
 - name: {{ include "common.names.fullname" . }}
   image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
   imagePullPolicy: {{ .Values.image.pullPolicy }}
+  {{- if .Values.image.args }}
```

Comment:
I'm thinking this should be .Values.args

what do you think at @bjw-s ?

Reply:
Agreed 👍

Reply:
Good call. I did ponder on that. Fixed!